commands.js

```diff
@@ -1217,7 +1217,8 @@ var commands = exports.commands = {
 
 		this.add('|raw|<div class="broadcast-blue"><b>' + target + '</b></div>');
 		this.logModCommand(user.name + " declared " + target);
-	},
+	}, 
+	htmldeclarehelp: ["/htmldeclare [message] - Anonymously announces a message with html codes. Requires: ~"],
```

Comment:
No, but you can get the list running in a PS window
`Object.keys(html4.ELEMENTS).filter(function (elem) {return html4.ELEMENTS[elem] | html4.eflags.UNSAFE})`